### DISCUSSION TITLE
Numeration feature

### DISCUSSION THREAD

**Discussion by Drotak** [flameshot-org/flameshot#2596] [Q&A] [answered]
It would be nice to have a numeration feature which would be a new tool. Inspired is this idea from greenshot, where you can add with just one click a circle with an increasing number in it. This is really helpful when you want to describe something to somebody. And you should also be able to increase the counter manually (when you are making several screenshot in a series you don't want to start at 1 again with each screenshot).
It is just an idea, because I loved this feature from greenshot, but I mainly use ubuntu now where it is not supported.

**Comment by borgmanJeremy** [comment, marked as answer]
Are you on a very old version? This was actually the first feature I added to flameshot when I joined the team years ago 😁

**Reply by Drotak** [reply]
That's very interesting. I really have a very old version, but I have installed it with apt on Ubuntu.
https://i.imgur.com/tOBTx93.png

Thank you for the heads up, I'll install it from the github repo now :)

**Reply by mmahmoudian** [reply]
@Drotak 
> but I have installed it with apt on Ubuntu

Ubuntu and other Debian-based distros usually don't have the latest version of any software. You can see the list of distros along with the Flameshot version they have in the following link:

https://repology.org/project/flameshot/versions

**Reply by Drotak** [reply]
@mmahmoudian Thank you for that clarification :)

**Reply by Drotak** [reply]
@borgmanJeremy I have installed now Version 11.0, but I'm still missing the feature to be able to manual increase the counter (for example in a tutorial series where I have a long text with multiple screenshots I don't want to start with 1 each time I make a screenshot because I want to continue my series)
Or have I overseen it?

**Reply by mmahmoudian** [reply]
@Drotak read the following:

- #1214
- #2162
